main.py

```diff
@@ -210,6 +212,7 @@ class Route:
   Route('/metrics/feature/timeline/popularity'),
   Route('/metrics/feature/timeline/popularity/<int:bucket_id>'),
   Route('/reports/spec_mentors'),
+  Route('/reports/review_latency'),
```

Comment:
I changed it to `review-latency`.

I don't think that we should change the spec_mentors in this PR because it is off-topic and because that URL is already live, so we would need to add a redirect or something.